src/components/Page/styles.module.css

```
@@ -0,0 +1,13 @@
+.page {
+    display: flex;
+    flex-grow: 1;
+
+    .leftPane {
+        display: flex;
+        flex-direction: column;
+        box-shadow: 0px 4px 4px 0px rgba(227, 227, 227, 0.25);
```

```suggestion
        box-shadow: 0px 4px 4px 0px rgba(227, 227, 227, 0.25);
```
Add design token here.

---

still need to finalize the token with the design team